magpie/ui/user/templates/edit_current_user.mako

```diff
@@ -30,23 +30,46 @@
 
 <h3>User Information</h3>
 
+<div class="alert alert-danger" id="EditService_UserSelfDeleteAlert">
+    <h3 class="alert-title-danger">Danger!</h3>
+    <div class="alert-info">
```

Comment:
I think one of the class needs to be adjusted here. 
The demo image shows a blue block area that should be red.

Reply:
Changed it to the red background, pics here :

![image](https://user-images.githubusercontent.com/36516122/101034387-78784f00-3548-11eb-8f4c-11aca630566a.png)

Reply:
Can you check also for the class that causes the "newline" of the text. 
The (!) should be on the same line followed by "Delete your account".

I think the problem is caused by same css class being reuse in 2 locations that require different configs.

Reply:
How does this look? (also added an extra space between the (!) and the text)

![image](https://user-images.githubusercontent.com/36516122/101053999-68676c00-3556-11eb-8749-d4256461edbe.png)